metadoc-cli/src/main/scala/metadoc/cli/MetadocCli.scala

```diff
@@ -48,16 +56,22 @@ object MetadocCli extends CaseApp[MetadocOptions] {
       .empty[String, d.Symbol]
       .withDefault(sym => d.Symbol(symbol = sym))
 
-    def add(name: Name): Unit = db.names.get(name.pos).foreach { symbol =>
-      val syntax = symbol.syntax
-      if (name.isBinder) {
-        symbols(syntax) =
-          symbols(syntax).copy(definition = Some(metadocPosition(name.pos)))
-      } else {
-        val old = symbols(syntax)
-        symbols(syntax) =
-          old.copy(references = old.references :+ metadocPosition(name.pos))
-      }
+    def add(name: Name): Unit = db.names.get(name.pos).foreach {
+      // do nothing for symbols that are not globally relevant.
+      case Symbol.Local(_) =>
+      case Symbol.Global(_, _: Signature.TypeParameter) =>
+      case Symbol.Global(_, _: Signature.TermParameter) =>
```

Comment:
I just noticed these changes break the definition provider for local symbols. We should be able to work around that by parsing the active model value instead storing them in the global index.